camo.py

```diff
@@ -14,14 +13,10 @@ def __init__(self, server, key):
     def image_url(self, url):
         return self.server + Image(url, self.key).path
 
-    def _is_absolute(self, url):
-        parsed_url = urlparse.urlparse(url.strip())
-        return bool(parsed_url.netloc)
-
     def _rewrite_url(self, url):
-        if url.startswith(self.server):
-            return url
-        elif not self._is_absolute(url):
+        # skip images that have already been proxied and
+        # images with data URIs
+        if url.startswith(self.server) or url.startswith("data:"):
```

Comment:
Are there other kinds of data urls besides `data:image` that we want to allow through?